apps/main/src/dex/components/PageRouterSwap/index.tsx

```diff
@@ -98,30 +97,54 @@ const QuickSwap = ({
 
   const isReady = pageLoaded && !isLoadingApi && isPageVisible
   const haveSigner = !!signerAddress
+
   const userFromBalance = userBalancesMapper[fromAddress]
   const userToBalance = userBalancesMapper[toAddress]
+
   const fromUsdRate = usdRatesMapper[fromAddress]
   const toUsdRate = usdRatesMapper[toAddress]
-  const fromToken = tokensNameMapper[fromAddress] ?? ''
-  const toToken = tokensNameMapper[toAddress] ?? ''
 
   const tokensMapperNonSmallTvlStr = useMemo(
     () => getTokensMapperStr(tokensMapperNonSmallTvl),
     [tokensMapperNonSmallTvl],
   )
 
-  const selectFromTokensList = useMemo(
-    () => getTokensObjList(selectFromList ?? selectToList, tokensMapper),
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-    [selectFromList, selectToListStr, tokensMapperStr],
+  function getTokensObjList(tokensList: string[] | undefined, tokensMapper: TokensMapper | undefined) {
```

Comment:
this should be in the global scope or be included in the `useMemo` dependencies, not sure why we don't get a warn